modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LabelTag.java

```diff
@@ -56,6 +56,10 @@ public boolean getLarge() {
 		return _large;
 	}
 
+	public boolean getTranslateLabel() {
```

Comment:
Naming is hard™ .  A standard naming convention for a boolean getter is to have `is` prefix, so this would be `isTranslateLabel`. This sounds awkward, grammatically incorrect, implying that a better name would an _adjective_ rather than an _action_. This convention makes a lot of sense, as an adjective can be true of false. For example, `disabled` is true or false. An attribute with action name implies that its value is implementation of that action, i.e. a function `disable()`.

In short, we should rename this attribute :slightly_smiling_face: . I'd suggest `translated`, with `isTranslated` getter. We can drop `label`, because tag has only one translatable string. Something like `translatedLabel` would maybe make sense in a different tag that has something like `translatedTitle`, and we want to provide API for translation opt-in on either.

On a related note, we can safely rename `getLarge` to `isLarge`.

Reply:
@markocikos I remember vaguely that we used to not used `is*` prefixes. Do you remember anything like that in the past? I feel like it came from bchan and the java side of formatting, but I don't fully remember. I did search through DXP and we do use it and I am personally not against using it but I remember being steered away from them in the past. 🤷

Reply:
When changing from `getLarge` -> `isLarge` and `getTranslated` -> `isTranslated` it requires a major version bump on `frontend-taglib-clay`. Considering it, does make sense just bump it? From https://github.com/liferay-frontend/liferay-portal/pull/1171#issuecomment-866411456 @nhpatt @bryceosterhaus @markocikos I'm not a pro in DXP semvar but in other packages it looks strong. Should I just place a deprecation notice in `getLarge` and see if semvar cries?

Reply:
@bryceosterhaus for recent tags ([Cards](https://github.com/liferay/liferay-portal/blob/4b9d2320ae42d5c58a56e0bced49f8fe4e7b7bbc/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/BaseCardTag.java#L141-L175), [Management Toolbar](https://github.com/liferay/liferay-portal/blob/4b9d2320ae42d5c58a56e0bced49f8fe4e7b7bbc/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java#L307-L399)), I know that we've used that `is*` (if you look at those methods: for cardsprefix and @markocikos 's explanation makes sense. As Marko said, sometimes it sounds correct (e.g. `isDisabled`) and sometimes it seems *totally* incorrect (e.g. `isShowCreationMenu`). I would think that "native english speakers" could be shocked about this but apparently it's not the case :shrug: 

@diegonvs I'd follow @markocikos 's advice - what I don't really like is marking `getLarge` as deprecated and adding `isLarge` which does exactly the same thing but we've already done that too in various places in order to follow this `is*` convention; I guess it's not that bad.

Reply:
> I remember vaguely that we used to not used `is*` prefixes. Do you remember anything like that in the past? I feel like it came from bchan and the java side of formatting, but I don't fully remember. I did search through DXP and we do use it and I am personally not against using it but I remember being steered away from them in the past.

@bryceosterhaus If I recall correctly, there is a bug in Soy related to getter prefix, when you wire Java to Soy. If you pass in a custom POJO as a param to Soy template, you can access it's boolean property only if it has a `get` prefix. I might be misremembering, I ran into this several years ago, in my LCS days, on DXP 7.0. This issue can be safely ignored for this PR. It is only applicable if you are creating a model POJO used by a Soy template.